public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/class-rest-feedback-controller.php

```diff
@@ -67,6 +68,36 @@ public function register_routes() {
 				),
 			)
 		);
+
+		// The Core users endpoint does not allow updating a user that is not a member of the site,
+		// so we have to create our own endpoint.
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/notifications/(?P<id>[\d]+)',
```

Comment:
I think this endpoint needs a separate controller, `class-rest-notifications-controller.php`, so that it can be its own base: `/wordcamp-speaker-feedback/v1/notifications/…`

It would also be nice to see tests for this endpoint (that doesn't have to block merge though)

Reply:
Sigh. You're right.

Moved to separate controller here: 40b5d8f

Tests added here: e836d56